tests/unittests/graphTest.cpp

```diff
@@ -460,3 +460,97 @@ TEST(Graph, disableUnrollingGroupConv) {
   EXPECT_GT(numberOfNodesOpenCL, numberOfNodesInterpreter);
 #endif // GLOW_WITH_OPENCL
 }
+
+/// Check that save nodes are properly scheduled.
+/// That is, they happen after the last use of the related variable.
+/// In that test, the order of the creation of the nodes give a valid schedule.
+TEST(Graph, schedulingOfSavesOrderProvided) {
+  ExecutionEngine EE;
+
+  Tensor inputs(ElemKind::FloatTy, {1, 32, 32, 3});
```

Comment:
nit: This seems to be unused

Reply:
Indeed! Interestingly the compiler didn’t complain about unused variable.